SU-SOE/soe_profile
# D8CORE-580: Add seckit

# READY FOR REVIEW

# Summary
- Enable SecKit at install time.

# Needed By (Date)
- Whenever this stack goes into production

# Criticality
- How critical is this PR on a 1-10 scale? 4/10
- ISO would like HSTS on all websites

# Steps to Test

1. Check out this branch
2. Create a new site
3. Request a page from your new site and ensure that the `strict-transport-security: max-age=31536000` header is set.
4. Run the included Behat test.

# Affected Projects or Products
- D8CORE, Jackalope

# Associated Issues and/or People
- JIRA ticket: D8CORE-580
- Other PRs:
    - https://github.com/SU-SWS/acsf-cardinalsites/pull/48
    - https://github.com/SU-SWS/stanford_profile/pull/55
    - https://github.com/SU-SWS/acsf-lelandd8/pull/4
    - https://github.com/SU-SWS/acsf-lelandd8/pull/8

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
